code/datums/supplypacks/misc_vr.dm

```diff
@@ -13,22 +13,42 @@
 	containername = "Belt-miner gear crate"
 	access = access_mining
 
+/datum/supply_packs/misc/mining_rig
+	name = "industrial hardsuit (empty)"
+	contains = list(
+			/obj/item/weapon/rig/industrial = 1
+			)
+	cost = 75
+	containertype = /obj/structure/closet/crate/secure/gear
+	containername = "industrial hardsuit crate"
+	access = access_mining
+
 /datum/supply_packs/misc/eva_rig
 	name = "eva hardsuit (empty)"
 	contains = list(
 			/obj/item/weapon/rig/eva = 1
 			)
-	cost = 150
+	cost = 125
 	containertype = /obj/structure/closet/crate/secure/gear
 	containername = "eva hardsuit crate"
-	access = access_mining
+	access = access_tech_storage
 
-/datum/supply_packs/misc/mining_rig
-	name = "industrial hardsuit (empty)"
+/datum/supply_packs/misc/medical_rig
+	name = "medical hardsuit (empty)"
 	contains = list(
-			/obj/item/weapon/rig/industrial = 1
+			/obj/item/weapon/rig/medical = 1
 			)
-	cost = 150
+	cost = 125
 	containertype = /obj/structure/closet/crate/secure/gear
-	containername = "industrial hardsuit crate"
-	access = access_mining
\ No newline at end of file
+	containername = "medical hardsuit crate"
+	access = access_medical
+
+/datum/supply_packs/misc/security_rig
+	name = "hazard hardsuit (empty)"
+	contains = list(
+			/obj/item/weapon/rig/hazard = 1
+			)
+	cost = 125
```

Comment:
I'm reasonably sure this thing is broken. Did someone fix the sprites?

Reply:
I tested them recently and it looked like they had no problems.